### DISCUSSION TITLE
Feature request: Dual pane editor

### DISCUSSION THREAD

**Discussion by Clock-Drift** [notaapp/talk#214] [Feature Requests]
Working with multiple documents at a time can quickly become frustrating, both due to heavily truncated file names in tabs slowing search as well as the lack of ability to have two tabs open side by side. The latter often makes it slow to use Nota to effectively collate research and write-up larger documents. The ability to either view two tabs side by side or to open a new editor would resolve the problem

**Comment by Auratelience** [comment]
Yes, this would be excellent.

**Comment by astoilkov** [comment]
We've discussed dual panes previously and we like the idea. It's a hard feature that will take time to implement but it's something we would like to do.

By "to open a new editor" do you mean open a new window? Thanks.

**Comment by Clock-Drift** [comment]
Yep a new window, that way we could have two tabs next to each other in separate windows (or two workspaces open at the same time) and cross reference

**Comment by astoilkov** [comment]
Got it. Thanks.

**Comment by jhildenbiddle** [comment]
Dual- or multi-pane would a welcome addition. 

I created a separate discussion (#156) for the feature request per the repo's `README.md` instructions. My intention wasn't to split the discussion between the two areas.

Thanks to @Clock-Drift for starting the discussion. :)

**Comment by petrbouchal** [comment]
Yes, this would be great - just commenting to document (my/a) use case: to have two documents open, one on each display.
